docs/tutorials/geomesa-quickstart-fsds.rst

```diff
@@ -0,0 +1,256 @@
+    02 719024963=719024963|||CITIZEN||051|6|2|6|4|City Of Sydney, New South Wales, Australia|AS|2018-01-01T00:00:00.000Z|POINT (151.217 -33.8833)
+    03 719025168=719025168|AUSTRALIAN|AUS|||051|18|1|10|4|Sydney, New South Wales, Australia|AS|2018-01-01T00:00:00.000Z|POINT (151.217 -33.8833)
+    04 719025178=719025178|AUSTRALIA|AUS|COMMUNITY||051|20|2|20|4|Sydney, New South Wales, Australia|AS|2018-01-01T00:00:00.000Z|POINT (151.217 -33.8833)
+    05 719025248=719025248|BUSINESS||||051|10|1|10|1|Australia|AS|2018-01-01T00:00:00.000Z|POINT (135 -25)
+    06 719025509=719025509|COMMUNITY||AUSTRALIA|AUS|051|2|1|2|1|Australia|AS|2018-01-01T00:00:00.000Z|POINT (135 -25)
+    07 719025555=719025555|DENMARK|DNK|||051|2|1|2|1|Australia|AS|2018-01-01T00:00:00.000Z|POINT (135 -25)
+    08 719025634=719025634|FIJI|FJI|||051|2|1|2|1|Fiji|FJ|2018-01-01T00:00:00.000Z|POINT (178 -18)
+    09 719025965=719025965|MIDWIFE||||051|10|1|10|4|Sydney, New South Wales, Australia|AS|2018-01-01T00:00:00.000Z|POINT (151.217 -33.8833)
+    10 719024909=719024909|||MELBOURNE|AUS|051|10|1|10|4|Melbourne, Victoria, Australia|AS|2018-01-01T00:00:00.000Z|POINT (144.967 -37.8167)
+
+    Returned 276 total features
+
+    Running query EventCode = '051' AND dtg DURING 2017-12-31T00:00:00+00:00/2018-01-02T00:00:00+00:00
+    Returning attributes [GLOBALEVENTID, dtg, geom]
+    01 719024909=719024909|2018-01-01T00:00:00.000Z|POINT (144.967 -37.8167)
+    02 719024963=719024963|2018-01-01T00:00:00.000Z|POINT (151.217 -33.8833)
+    03 719025168=719025168|2018-01-01T00:00:00.000Z|POINT (151.217 -33.8833)
+    04 719025178=719025178|2018-01-01T00:00:00.000Z|POINT (151.217 -33.8833)
+    05 719025248=719025248|2018-01-01T00:00:00.000Z|POINT (135 -25)
+    06 719025509=719025509|2018-01-01T00:00:00.000Z|POINT (135 -25)
+    07 719025555=719025555|2018-01-01T00:00:00.000Z|POINT (135 -25)
+    08 719025634=719025634|2018-01-01T00:00:00.000Z|POINT (178 -18)
+    09 719025965=719025965|2018-01-01T00:00:00.000Z|POINT (151.217 -33.8833)
+    10 719024909=719024909|2018-01-01T00:00:00.000Z|POINT (144.967 -37.8167)
+
+    Returned 276 total features
+
+    Done
+
+
+Looking at the Code
+-------------------
+
+The source code is meant to be accessible for this tutorial. The main logic is contained in
+the generic ``org.geomesa.example.quickstart.GeoMesaQuickStart`` in the ``geomesa-quickstart-common`` module,
+which is datastore agnostic. Some relevant methods are:
+
+-  ``createDataStore`` get a datastore instance from the input configuration
+-  ``createSchema`` create the schema in the datastore, as a pre-requisite to writing data
+-  ``writeFeatures`` use a ``FeatureWriter`` to write features to the datastore
+-  ``queryFeatures`` run several queries against the datastore
+-  ``cleanup`` deletes data and disposes of the datastore instance. Note the FSDS is not able to automatically cleanup at this time.
```

Comment:
i'll update the other tutorials to use consistent tense - can you do the same here?

Reply:
Ah, good catch, forgot about that

Reply:
maybe change to "delete the sample data and dispose of the ..."